recipes/symengine/all/conanfile.py

```diff
@@ -34,11 +39,47 @@ class SymengineConan(ConanFile):
     }
     short_paths = True
 
+    @property
+    def _min_cppstd(self):
+        return 11
+    
+    @property
+    def _minimium_compilers_version(self):
+        return {
+            # Can't compile with earlier versions even if C++11 supported
+            "gcc": "7"
+        }
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+    
+    def validate(self):
+        check_min_cppstd(self, self._min_cppstd)
+        
+        minimum_version = self._minimium_compilers_version.get(str(self.settings.compiler))
+        if minimum_version and Version(self.version) < minimum_version:
+            raise ConanInvalidConfiguration(f"{self.ref} requires {self.settings.compiler} >= {minimum_version}")
```

Comment:
Thank you for approving! I am now wondering if this is correct? When I do
```
conan install --requires symengine/0.13.0@ --build=missing
```
I get the error:
```
symengine/0.13.0: Invalid: symengine/0.13.0 requires gcc >= 7
```
But I have `compiler.version=13` in my settings. Is `self.version` the right thing to compare here?

Reply:
https://github.com/conan-io/conan-center-index/pull/25884